android/java/res/layout/binance_deposit_layout.xml

```diff
@@ -67,7 +67,7 @@
 			        android:layout_gravity="center_vertical"
 			        android:contentDescription="@null"
 			        android:padding="8dp"
-			        android:src="@drawable/qrcode_dark"/>
+			        android:src="@drawable/ic_qr_code"/>
```

Comment:
We should use app:srcCompat with vector drawables.

Reply:
Fixed.